funcframework/events.go

```diff
@@ -126,7 +126,9 @@ func getBackgroundEvent(body []byte, path string) (*metadata.Metadata, interface
 	if possible.BackgroundEvent == nil && possible.LegacyPushSubscriptionEvent != nil {
 		topic, err := pubsub.ExtractTopicFromRequestPath(path)
 		if err != nil {
-			fmt.Printf("WARNING: %s", err)
+			replacer := strings.NewReplacer("\n", "", "\r", "")
```

Comment:
Based on the alert's stack trace, we should probably do this on line 53 of pubsub.go and apply it just to the path.

We probably don't want to remove whitespaces everywhere in the error message as something in our own code might have added it to make the error message more readable.